strawberryfields/backends/fockbackend/backend.py

```diff
@@ -50,8 +50,8 @@ def _remap_modes(self, modes):
         submap = [map_[m] for m in modes]
```

Comment:
(Throughout): Before merging, let's confirm that these still import correct docstrings in compiled docs